pypeit/core/flux_calib.py

```diff
@@ -1190,13 +1197,13 @@ def standard_zeropoint(wave, Nlam, Nlam_ivar, Nlam_gpm, flam_true, mask_balm=Non
       counts/s/Angstrom as observed
     Nlam_ivar : `numpy.ndarray`_
       inverse variance of counts/s/Angstrom
+    Nlam_gpm : `numpy.ndarray`_
+      mask for bad pixels. True is good.
     flam_true : Quantity array
       standard star true flux (erg/s/cm^2/A)
-    msk_bad : `numpy.ndarray`_
-      mask for bad pixels. True is good.
-    msk_star: `numpy.ndarray`_
+    mask_balm: `numpy.ndarray`_
```

Comment:
Thanks for cleaning this masking stuff up. It was a constant source of headache.